accounts/management/commands/sync_firebase_user.py

```diff
@@ -0,0 +1,56 @@
+import environ
+from django.core.management.base import BaseCommand
+import requests
+
+from accounts.models import User
+
+env = environ.Env()
+
+FIREBASE_VERIFY_PASSWORD_URL = (
+    "https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword"
+)
+
+
+class Command(BaseCommand):
+    help = "Syncs a Firebase user with a Django user"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "email",
+            type=str,
+            nargs=1,
+            help="Firebase & Django account email",
+        )
+        parser.add_argument(
+            "password",
+            type=str,
+            nargs=1,
+            help="Firebase account password",
+        )
+
+    def handle(self, *args, **options):
+        email = options.get("email")[0]
+        password = options.get("password")[0]
+
+        json = requests.post(
+            f"{FIREBASE_VERIFY_PASSWORD_URL}?key={env.str('FIREBASE_WEB_API_KEY')}",
+            data={
+                "email": email,
+                "password": password,
+                "returnSecureToken": True,
+            },
+        ).json()
+
+        firebase_uid = json["localId"]
+        token = json["idToken"]
+
+        user = User.objects.get(email=email)
```

Comment:
itd be nice if we added some exception handling, but if it's just being used by us then it'll be fine

Reply:
yeah it'll just be us using it - I think it'd help us diagnose seeing the real error if something does go wrong